src/pyrokinetics/local_geometry/local_geometry.py

```diff
@@ -429,14 +519,25 @@ def get_bunit_over_b0(self):
 
         theta = np.linspace(0, 2 * pi, 256)
 
-        R, Z = self.get_flux_surface(theta=theta, normalised=True)
+        R, Z = self.get_flux_surface(theta=theta)
+
+        # Roll doesn't work on pint quantities...
```

Comment:
Could add `TODO` with pint version number, so this could be found and fixed at a later date?